src/source/Main.scala

```diff
@@ -16,14 +16,15 @@
 
 package djinni
 
-import java.io.{IOException, FileInputStream, InputStreamReader, File, BufferedWriter, FileWriter}
+import java.io.{IOException, FileNotFoundException, FileInputStream, InputStreamReader, File, BufferedWriter, FileWriter}
 
 import djinni.generatorTools._
 
 object Main {
 
   def main(args: Array[String]) {
     var idlFile: File = null
+    var idlIncludePaths: List[String] = List(".")
```

Comment:
"." doesn't seem like the right default here.  I feel like the default here should be empty, while the hard-coded default should be the path of the file containing the @import.  However "." on the command-line should mean the current working directory, which may not be the same.